### DISCUSSION TITLE
Enhancing Product Dissemination for Unskilled Users

### DISCUSSION THREAD

**Discussion by ckirilov** [sqlpage/SQLPage#535] [Ideas]
Hi everyone,

SQLPage is an interesting idea and a promising product, but I believe it needs better dissemination and more comprehensive tutorials. Researchers in the humanities could be a significant user group, but they require more resources than what is currently available.

SQLPage is advertised as very user-friendly, and it certainly is. However, a few YouTube tutorials demonstrating the creation of a website from start to finish would be extremely beneficial. At present, there is only a recorded talk and a semi-tutorial, which are not particularly informative.

Trust me, you have a huge potential user base. As a humanitarian myself, I have no issues with SQLPage, but many in the humanities are not familiar with SQL, let alone how a SQLite (or any other) database works. While they may understand databases conceptually and can define the right cardinalities, they often have no idea how to implement or interact with a database. They typically rely on programmers to build custom applications, but there are not many programmers willing to work wholeheartedly with underfunded research institutions.

I wish you all the best and hope this great product continues to improve.

**Comment by lovasoa** [comment]
Hi @ckirilov !

  @alexisrc1 has recently started [a YouTube video series](https://m.youtube.com/playlist?list=PLTue_qIAHxAf9fEjBY2CN0N_5XOiffOk_) about SQLPage. Could you have a look and give your feedback?

**Reply by alexisrc1** [reply]
All the files used in the tutorials are available [here](https://drive.google.com/drive/folders/13m3vUejJVKo9DrSU6yGLC3OUpPEzLcCD?usp=sharing). Let us know what you think! More videos are coming soon, explaining some of the key concepts in detail.
